doc/source/parameters/dem/model_parameters.rst

```diff
@@ -27,16 +27,18 @@ In this subsection, contact detection, force models, time integration, load bala
       set frequency               = 100000 # if method = frequent
       set dynamic check frequency = 10000  # if method = dynamic
       set threshold               = 0.5    # if method = dynamic
-    end
 
     # Particle-particle contact force model
-    # Choices are linear|hertz_mindlin_limit_overlap|hertz_mindlin_limit_force|hertz|hertz_JKR
+    # Choices are linear|hertz_mindlin_limit_overlap|hertz_mindlin_limit_force|hertz|hertz_JKR|DMT
     set particle particle contact force method = hertz_mindlin_limit_overlap
 
     # Particle-wall contact force model
-    # Choices are linear|nonlinear|JKR
+    # Choices are linear|nonlinear|JKR|DMT
     set particle wall contact force method = nonlinear
 
+    # DMT cut-off threshold
+    set DMT cut-off threshold = 0.1
```

Comment:
Maybe I would add a check related to the neighborhood threshold parameter since its value is now a bit dependent on that cut-off threshold if you don't want to miss a long distance force.